### ISSUE TITLE
Не скачивает мангу

### ISSUE THREAD

**Issue by panchenko9b** [4ndv/mrmanga#9]
На ридманге вот такое вылазит при скачке. С минтманги качает, нормально, но есть одно НО, о нем ниже.
![image](https://user-images.githubusercontent.com/44533357/77915077-89790e00-729f-11ea-873c-91bc30fa526c.png)
 
Вот, все файлы с рида и минта выглядят так, соответственно.
![image](https://user-images.githubusercontent.com/44533357/77915165-a7df0980-729f-11ea-90dc-5c5546ee1df0.png)
![image](https://user-images.githubusercontent.com/44533357/77915174-add4ea80-729f-11ea-9f33-24c9f5cd1021.png)

**Comment by panchenko9b** [comment]
Too many requests from your computer
также потом не пускает на ридмангу, после ошибки в загрузчике, с вот таким сообщением

**Comment by 4ndv** [comment]
> Too many requests from your computer

Выглядит как бан от ридманги, подозреваю, что и ошибка в загрузчике по этой же причине, нужно смотреть чего поменяли

**Comment by 4ndv** [comment]
Судя по всему, добавилась какая-то подпись ссылок, нужно смотреть принцип по которому она вычисляется

**Comment by 4ndv** [comment]
Ни по какому, ссылки просто передаются в в теле страницы, просто есть разные ссылки на разные размеры ¯\_(ツ)_/¯ 

Нужно будет переделать, но пока нет времени

**Comment by panchenko9b** [comment]
Такая проблема только у меня? Или других жалоб нет, потому что никто не пользуется программой?

**Comment by 4ndv** [comment]
Нет, проблема общая, до сих пор не дошли руки что либо посмотреть, ну и сам я перешел на мангалиб, так что особого желания заново всё это ковырять пока что нету
